MyFirstWinRtc/Microsoft.WinRTC.Simple.VideoConferencing/pch.h

```diff
@@ -3,6 +3,11 @@
 
 #pragma once
 
+//Silencing the C++17 depreciated feature that causing visual studio to error during Build.
```

Comment:
How about: "abseil is using some C++ features that were deprecated in C++17. We build with warning as errors. The following preprocessor variables make STL not emit the warning"

Reply:
thanks for the contribution. its has been incorporated